src/web/components/AuditTrail/AuditTrail.scss

```diff
@@ -0,0 +1,73 @@
+.audit-trail-table-container {
+  .audit-log-table {
+    font-size: 0.875rem;
+    width: 100%;
+
+    thead {
+      font-weight: 700;
+      border: none;
+    }
+
+    td {
+      padding-top: 15px;
```

Comment:
We could probably add 20px of padding here to space the columns out a bit better, i.e.
`padding: 15px 20px 15px 0;`
![image](https://github.com/user-attachments/assets/de3f1af3-a709-439c-bcef-1837fe838172)